src/components/CommentCard.jsx

```diff
@@ -1,11 +1,30 @@
-import React from 'react';
+import React, { useContext, useState } from 'react';
+import { deleteComment } from '../api';
+import { UserContext } from '../User';
 
-export default function CommentCard({ comment }) {
+export default function CommentCard({ comment, setComments }) {
   const { author, body, comment_id, created_at, votes } = comment;
 
+  const [confirmDelete, setConfirmDelete] = useState(false);
+
+  const { user } = useContext(UserContext);
+
   const parsedDate = new Date(created_at);
   const readableDate = String(parsedDate).slice(0, 24);
 
+  const handleDelete = () => {
+    setComments((prevComments) => {
+      return prevComments.filter((comment) => {
+        return comment.comment_id !== comment_id;
+      });
+    });
+    deleteComment(comment_id).catch(() => {
+      setComments((prevComments) => {
+        return [...prevComments, comment];
+      });
```

Comment:
UX wise- this is confusing. 
It doesnt delete it, but instead just adds it to the bottom of the list. 
If im at the top, it looks like its gone, but actually its not deleted and is now at the bottom. 

Also, we'd ideally need some feedback to let us know its no been deleted successfully.